src/components/EventBanner/EventBanner.tsx

```diff
@@ -18,9 +18,9 @@ const EventBanner = ({ isMVMFEnabled }: Props) => {
         </span>
       </div>
       <div className="event-banner-cta">
-        <Button className="cta" size="medium" href="https://decentraland.org/blog/announcements/">
+        <a className="cta ui button medium" href="https://decentraland.org/blog/announcements/" target="_blank" rel="noopener noreferrer">
```

Comment:
```suggestion
<Button
                className="cta"
                size="medium"
                as={'a'}
                target="_blank"
                rel="noopener noreferrer"
                href={'link'}
              >
                {t('event_banner.small.cta')}
              </Button>
```
I think this one looks a bit cleaner, wydt?

Reply:
thanks for the feedback @juanmahidalgo! yeah, it looks better